### DISCUSSION TITLE
Slingbox Pro HD

### DISCUSSION THREAD

**Discussion by BroozerL** [GerryDazoo/Slinger#662] [Q&A]
Hi - after some fiddling with port redirection, I'm finally able to connect to the Slinger Server I've installed/configured on my Windows 10 PC and view OTA on the tuner on my HD Pro.  Thank you Gerry for providing a way to repurpose the boxes.

My question is:
Is it possible to perform a channel rescan on the built-in tuner?  If so, how?

**Comment by bradut** [comment]
Hi @BroozerL 
Yes, channel re-scanning is possible via SlingPlayer 1.5.x, as explained in the pdf file that describes the entire resuscitating process.
I am specifically referring to scanning analog channels.

**Comment by BroozerL** [comment]
Hi Gerry,

Thanks for the reply.  Of course, with the exception of perhaps a mere 
handful of low-powered translators, all OTA signals in the US are now 
digital.  To the best of my knowledge, everything in metro Atlanta is 
now digital.

I dropped cable TV about 9 years ago and currently have no TV 
subscriptions, e.g., cable, satellite, streaming, etc. and am interested 
only in OTA placeshifting.

Is there any way to scan for digital channels on the ProHD tuner?  
Otherwise, we're looking at attaching a digital converter box, which we 
can place on any Slingbox which will work with the converter box and 
your server.

Best,
Bruce

---

-------------------------

Bruce Lipsitz, Atlanta, GA  USA

On 2024-08-19 12:58, Bradut Dima wrote:

> Hi @BroozerL [1]
> Yes, channel re-scanning is possible via SlingPlayer 1.5.x, as 
> explained in the pdf file that describes the entire resuscitating 
> process.
> I am specifically referring to scanning analog channels.
> 
> --
> Reply to this email directly, view it on GitHub [2], or unsubscribe 
> [3].
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>


Links:
------
[1] https://github.com/BroozerL
[2] 
https://github.com/GerryDazoo/Slinger/discussions/662#discussioncomment-10386612
[3]

**Reply by bradut** [reply]
Hi @BroozerL,

Apologies for my very late reply.

I initially noticed that you had replied to Gerry, so I held off on responding, only to realize today that you were actually replying to my message. (I am not Gerry.)

My previous reply about analog channels applies specifically to cable providers that offer both digital and analog subscriptions. Regarding antenna / OTA, I think that in most countries this is only digital.
One advantage of analog channels is that the Slingbox can switch channels without needing to simulate a remote control to send IR signals.

Yes, it is possible to scan digital channels with the Slingbox Pro HD. 
In this case, switching channels requires the Slingbox Pro HD to act as a remote control, sending infrared signals to your cable provider’s TV box. 
During setup, you’ll need to input the type of your TV box, as described in the setup PDF.

**Things I’m unsure about—maybe check other discussions:**
Given that _you are connecting directly to an antenna,_ meaning the Slingbox should send signals directly to your TV set, without involving a TV box. I’m not sure if the setup process will provide options for this type of connection, but I hope it does.

If you decide to use a digital converter:
- If the converter outputs _all channels_ as analog, then you won’t need to simulate a remote control; the Slingbox can handle this setup independently.
- However, if the converter requires a remote control to switch channels, you’ll need to configure the Slingbox to send IR signals to the converter. I’m not certain how the setup will accommodate this.

I believe others have connected directly to a digital antenna, so it may be helpful to check some of the related discussions.
(Such as **this discussion https://github.com/GerryDazoo/Slinger/discussions/554**)

**Reply by RayMan412** [reply]
I was able to get the ProHD Scanned with a MAC
The MAC version allows you to do this, but the MAC version's certificate expired. You need a workaround and I got it to work.

PRO HD - DIGITAL TUNING WITH MAC SLINGPLAYER 

Once you have unbricked the Slingbox, you can then do this:

I used Slingplayer 1.0.10.dmg
https://www.mediafire.com/file/l53odja366hvmm7/SlingPlayer-1.0.10.dmg/file


When you try to run it, you get the “no longer allowed”
Here was how I was able to get it bypassed:

For anyone fighting this issue "Later versions of Apple OS will not let you run the older .DMG images after install IE: "Move it to Trash or Cancel..." Slingplayer.app versions......here you go:


(CLI - Mac Command Line Interface)


From the CLI....sudo -i to root.
As root:1. root# ls -al@



This will show you all quarantined installed applications and if Slingplayer.app was quarantined by Apple!


(NOTE - I didn’t see it in there, but I went ahead and changed the directory and just did the following)



2. To remove the quarantine:

3. Note: (you need to be sitting in the Applications directory and it needs to be done recursively)

As root: root# xattr -cr Slingplayer.app


Reboot and test.




Lots of good links here:
http://www.hifi-remote.com/forums/viewtopic.php?p=145528#145528
